MC/config/PWGHF/pythia8/generator/pythia8_beauty_with_mudecays_Mode2.cfg

```diff
@@ -0,0 +1,71 @@
+### authors: Fabrizio Grosa (fabrizio.grosa@cern.ch)
+###          Cristina Terrevoli (cristina.terrevoli@cern.ch)
+###          Fabio Catalano (fabio.catalano@cern.ch)
+###          Nicole Bastid (nicole.bastid@cern.ch)
+### Beauty decay muons 
+### last update:  November 2024
+
+### beams
+Beams:idA 2212			# proton
+Beams:idB 2212 			# proton
+Beams:eCM 13600. 		# GeV
+
+### processes
+SoftQCD:inelastic on		# all inelastic processes
+
+### decays
+ParticleDecays:limitTau0 on	
+ParticleDecays:tau0Max 10.
+
+### switching on Pythia Mode2
+ColourReconnection:mode 1
+ColourReconnection:allowDoubleJunRem off
+ColourReconnection:m0 0.3
+ColourReconnection:allowJunctions on
+ColourReconnection:junctionCorrection 1.20
+ColourReconnection:timeDilationMode 2
+ColourReconnection:timeDilationPar 0.18
+StringPT:sigma 0.335
+StringZ:aLund 0.36
+StringZ:bLund 0.56
+StringFlav:probQQtoQ 0.078
+StringFlav:ProbStoUD 0.2
+StringFlav:probQQ1toQQ0join 0.0275,0.0275,0.0275,0.0275
+MultiPartonInteractions:pT0Ref 2.15
+BeamRemnants:remnantMode 1
+BeamRemnants:saturation 5
+
+# Correct decay lengths (wrong in PYTHIA8 decay table)
+# Lb
+5122:tau0 = 0.4390
+# Xic0
+4132:tau0 = 0.0455
+# OmegaC
+4332:tau0 = 0.0803
+
+
+### switch off all decay channels for charm
+511:onMode = off
+521:onMode = off
+531:onMode = off
+5122:onMode = off
+5132:onMode = off
+5232:onMode = off
+5332:onMode = off
+
+###Semimuonic decays of charm
+
+### B0 -> mu X
+511:onIfAny = 13
+### B+/- -> mu X
+521:onIfAny = 13
+### B_s -> mu X
+531:onIfAny = 13
+### Lambda_b -> mu X
+5122:onIfAny = 13
+### Sigma_b -> mu X
+5132:onIfAny = 13
+### Sigma_b0 -> mu X
+5232:onIfAny = 13
```

Comment:
Sigma_b state does not decay to leptons, since it decays strongly to Lambda_b (https://pdglive.lbl.gov/Particle.action?init=0&node=S026&home=BXXX045)